pos_event_sale/models/pos_config.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Camptocamp SA - Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    iface_event_sale = fields.Boolean("Sell Events")
```

Comment:
maybe a default to True ? we can consider that if people are installing the module, they want the feature ? (as other pos modules...)

Reply:
I pushed the changes. Although I agree with you, this can have undersired effects during tests if we consider this to be the norm.. (see https://github.com/OCA/pos/pull/642 for example)

This module is only adding to core features, not reducing them.. so it shouldn't be an issue